src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  padding: 5px 50px;
+  background: #FFFFFF;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.links li {
+  display: inline-block;
+}
+
+.links li a {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  line-height: 14px;
+  font-size: 12px;
+}
+
+.nav_link {
+  color: #000000;
+  padding: 10px;
+  text-decoration: none;
+  position: relative;
+  animation: all 0.2s ease 0s;
+}
+
+.nav_link:hover, .nav_link:focus {
+  color: #00ACDC;
+}
+
+.nav_link:active {
+  color: #00ACDC;
+}
+
+.nav_link:before {
+  content: "";
+  position: absolute;
+  bottom: -10px;
```

after using height for link with 0 should work

```suggestion
  bottom: 0;
```